release-tools/build.make

```diff
@@ -74,7 +74,7 @@ endif
 # separated by semicolon. An empty variable or empty entry (= just a
 # semicolon) builds for the default platform of the current Go
 # toolchain.
-BUILD_PLATFORMS =
+BUILD_PLATFORMS = linux amd64
```

Comment:
Not sure if these are equatable! `linux` is an OS and `amd64` a processor architecture!
@hairyhum Could you please comment?